Dockerfile

```diff
@@ -3,3 +3,5 @@ FROM rocker/binder:3.4.2
 # Run install.r if it exists
 RUN if [ -f install.R ]; then R --quiet -f install.R; fi
 
+COPY . .
```

Comment:
Does this really work? Copying the current folder to the current folder? I thought we had to move stuff to $HOME

Reply:
Works when running repo2docker locally ... I can change for $HOME.